src/System.Linq/tests/MinMaxTests.cs

```diff
@@ -0,0 +1,373 @@
+﻿using System.Collections.Generic;
```

Comment:
Thank you for adding more tests.  Could these be done as a separate PR to go in first?